File: BAEKJOON/Node/14000/14900/14913.js
const fs = require("fs")
const filepath = process.platform === "linux" ? "/dev/stdin" : "./input.txt"
const input = fs.readFileSync(filepath).toString().trim()

const [a, d, k] = input.split(" ").map(x => parseInt(x))
const diff = k - a
const q = diff / d
const r = diff % d

if (r !== 0 || q < 0) {
  console.log("X")
} else {
  console.log(q + 1)
}

File: BAEKJOON/Python/14000/14900/14913.py
"""
Title : 등차수열에서 항 번호 찾기
Link : https://www.acmicpc.net/problem/14913
"""

from sys import stdin

input = stdin.readline


if __name__ == "__main__":
    a, d, k = map(int, input().split())
    diff = k - a
    if diff * d < 0 or diff % d:
        print("X")
    else:
        print(diff // d + 1)
